public/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,600,700,800">
+    <link rel="stylesheet" href="http://www.material-ui.com/css/font-icons/style.css">
```

Comment:
Hmm, I see. This css is only for the Github icon. So either replace the icon with a [Material icon](https://material.io/icons/) or just download the stylesheet file and save it to our folder. The second option is probably better, as you say, because with the Github icon it is pretty clear what the button does.